lib/server.js

```diff
@@ -57,6 +57,25 @@ function create(log, error, config, routes, db) {
     }
   }
 
+  function makeVerifiedCredentialFn (dbGetFn) {
+    return function (id, cb) {
+      log.trace({ op: 'DB.getToken', id: id })
+      if (!HEX_STRING.test(id)) {
+        return process.nextTick(cb.bind(null, null, null)) // not found
+      }
+      dbGetFn(Buffer(id, 'hex'))
+        .done(
+          function (token) {
+            if (! token.tokenVerified) {
+              return cb(error.unknownAccount())
```

Comment:
I believe we'll need to throw `error.unverifiedAccount` here in order to trigger the correct behaviour in clients (particularly android, which checks for account verification by polling `/account/keys` until it gets a successful response)